exporter/opencensusexporter/package_test.go

```diff
@@ -0,0 +1,14 @@
+// Copyright The OpenTelemetry Authors
```

Comment:
Let me know if there's a preference to move this out to a separate PR, happy to do whatever is most clear here.